buildSrc/src/main/kotlin/Utils.kt

```diff
@@ -0,0 +1,19 @@
+object Projects {
+    const val GROUP = "team.backend"
+    const val VERSIONS = "0.0.1-SNAPSHOT"
+}
+
+object Modules {
+    const val CORE = ":core"
+    const val API = ":api"
+    const val CLIENT = ":client"
+}
+
+object CompilerOptions {
+    const val NULL_SAFETY = "-Xjsr305=strict"
+}
+
+object Resources {
+    const val SOURCE_PATH = "../chaterview-private"
```

Comment:
I think `YML_PATH` or `PRIVATE_YML_PATH` is more appropriate than the name `SOURCE_PATH`.

Reply:
I think it would be better to reconsider the name later. It seems less likely that something else will be added at this stage.

If xml is added in addition to yml, the name must be changed again.

```
from(Resources.SOURCE_PATH) {
   include("*.yml")
}
```